include/aos/common/pkcs11/privatekey.hpp

```diff
@@ -132,11 +132,6 @@ class PKCS11ECDSAPrivateKey : public crypto::PrivateKeyItf {
     }
 
 private:
-    Error ParseECDSASignature(const Array<uint8_t>& src, Array<uint8_t>& r, Array<uint8_t>& s) const;
```

Comment:
Commit message: better to write: remove asn1 encoding or similar.

Reply:
ok I will change it.